client/src/api/userConfiguration.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * @category Client
+ * @module
+ */
+
+/**
+ * @description Get the current user id
+ */
+const getCurrentUserId = async () => {
```

Comment:
In no case a user Id would be of use, if it needed to be retrieved from the backend. A userID should be used in th backend to create callouts TO the backend and even maybe to authorize to requests if they are based on the user (only retrieve those fields who are allowed for that user).
For our test purposes it would be of use to change that userID, but this should 100% happen in the frontend. No backend communication should be needed or is necessary for this. Please adjust the code to allow for this.

Reply:
I partly agree and thats why I implemented it frontend only at first. The reason we have that backend call now is because one of the todos of that user story was: "the current value of the field should be persisted so that the same user is logged in even when the app is restarted.". I don't see another way to persist the current value without saving it on the server side. I agree it is kind of ugly but as you said it is for testing purposes only so I figured it would be allright.

Reply:
Please use the session storage for this. We already use this in i.e. client/src/components/content/PropertiesPanel/ProprtiesPanelView/PropertiesPanelView.jsx in line 68

Reply:
Thanks for the hint! I just implemented it with the session storage and it is way better now :)